docker/README.md

```diff
@@ -1,15 +1,16 @@
-Tile Server Docker
-==================
-
+# TileServer Docker Config #
 To build the image, run:
+    `docker build -t spandex-http .`
 
-    docker build --rm -t tileserver .
-
-Optional environment variables:
+Or, if you want to replace old versions:
+    `docker build --rm -t spandex-http .`
 
-Tileserver requires the environment variable ZOOKEEPER_ENSEMBLE to be
-set with the syntax:
+## Required Environment Variables ##
+* `ZOOKEEPER_ENSEMBLE` - A list of hostnames and ports of zookeeper instances. eg: ["10.0.0.1:2181", "10.0.0.2:2818"]
 
-["10.0.0.1:2181", "10.0.0.2:2818"]
+## Optional Runtime Variables ##
+See the [DockerFile](Dockerfile) for defaults.
 
-TILESERVER_MAXMEM     (default 2048m)  
+* `JAVA_XMX`                - Sets the JVM heap size.
+* `MIN_THREADS`             - Sets the minimum number of server threads.
+* `MAX_THREADS`             - Sets the maximum number of server threads.
```

Comment:
I'd add a note about limiting max threads based on memory use.  ie. why do we have this strange tiny default.